### PR TITLE
Basic FAQ and Changing Integration Test Framework pages

### PR DESCRIPTION
**Pull request by melchiork**
### 🤔 What's changed?

Documentation only changes. Basic FAQ and Changing Integration Test Framework pages.

### ⚡️ What's your motivation? 

Everybody likes to have docs nobody likes to write them.

### 🏷️ What kind of change is this?

- :book: Documentation (improvements without changing code)

### ♻️ Anything particular you want feedback on?

Should I aim to rewrite SpecFlow FAQ? It does not look very focused.

### 📋 Checklist:

- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/guides/how-to-change-test-execution-framework.md
Comment (clrudolphi) [inline docs/guides/how-to-change-test-execution-framework.md:1]: Shouldn't the line:
>To change test execution framework reference relevant NuGet SpecFlow pacakge. For details see:

instead be:

>To change test execution framework reference relevant NuGet Reqnroll package. For details see:

(two changes, SpecFlow->Reqnroll and correct spelling of package)

**Review by clrudolphi — COMMENTED** [review]
Thanks for taking the time to improve Reqnroll!

File: docs/guides/how-to-change-test-execution-framework.md
Comment (melchiork) [reply docs/guides/how-to-change-test-execution-framework.md:1]: I always promise myself to spellcheck with a tool and not by looking hard on what I wrote and I usually fail.
Fixed.

**Comment by melchiork** [comment]
@clrudolphi Anything else you'd suggest to change or improve?

File: docs/guides/how-to-change-test-execution-framework.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # How to change the test execution framework used by Reqnroll
 
-% TODO
-```{admonition} Documentation is in progress
-:class: warning
+To change test execution framework reference relevant NuGet Reqnroll package. For details see:
```
Comment (clrudolphi) [inline docs/guides/how-to-change-test-execution-framework.md:3]: Suggested grammar change:
To change the test execution framework, reference the relevant NuGet Reqnroll package.

Also - may want to also mention that the test framework specific attributes, such as [Test] or [Theory] also need to be changed.

**Comment by clrudolphi** [comment]
> @clrudolphi Anything else you'd suggest to change or improve?

Please add an entry in the CHANGELOG.md (near the top, follow the template) so that these additions are highlighted in the next release.

Other than the minor items just mentioned, these look good. Please add Gaspar as a reviewer to this PR. 
I will mark this PR with my approval, but Gaspar is the one to merge it into Main.

File: docs/guides/how-to-change-test-execution-framework.md (hunk 1 above)
Comment (melchiork) [reply docs/guides/how-to-change-test-execution-framework.md:3]: Why would we need to change any attribute when all code using those is generated in the background?

Comment (melchiork) [reply docs/guides/how-to-change-test-execution-framework.md]: Grammar fixed, thanks.

**Comment by melchiork** [comment]
> > @clrudolphi Anything else you'd suggest to change or improve?
> 
> Please add an entry in the CHANGELOG.md (near the top, follow the template) so that these additions are highlighted in the next release.
> 
> Other than the minor items just mentioned, these look good. Please add Gaspar as a reviewer to this PR. I will mark this PR with my approval, but Gaspar is the one to merge it into Main.

Sure, @gasparnagy if you could review it would be grand.

File: docs/guides/how-to-change-test-execution-framework.md (hunk 1 above)
Comment (clrudolphi) [reply docs/guides/how-to-change-test-execution-framework.md]: > Why would we need to change any attribute when all code using those is generated in the background?

'Doh!  I was thinking of something else. Ignore that comment.

**Comment by melchiork** [comment]
> This is super, thx! My only request is to undo the changes in CHANGELOG.md: documentation changes are immediately become publicly available and they are not bound to releases or release versions, so we don't include them in the changelog.

Done.

**Review by gasparnagy — APPROVED** [review]
thx!

**Pull request merged**